src/platform_impl/linux/event_loop.rs

```diff
@@ -72,13 +72,21 @@ impl<T> EventLoopWindowTarget<T> {
   }
 }
 
+enum EventState {
+  NewStart,
+  EventQueue,
+  DrawQueue,
```

Comment:
might be worth documenting what they mean.

Reply:
You are alright. Give me one more day. I'm gonna draw some sweet chart in the comment to demonstrate how the state flows.

Reply:
Wow, that would be brilliant.